src/components/BookList.jsx

```diff
@@ -0,0 +1,17 @@
+import { useSelector } from 'react-redux';
+import AddBook from './AddBook';
+import Book from './Book';
+
+export default function BookList() {
+  const bookList = useSelector((state) => state.booklist.value);
+
+  return (
+    <section className="books-container-section">
+      {bookList.map((book) => (
+        <Book key={book.id} title={book.title} author={book.author} id={book.id} />
+      ))}
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/233161538-33072b43-eeed-407f-b0b0-ef62ff2b80cf.png)

- [ ] It would be very nice if you remove the live demo link from the table of contents because it is not working because you don't have a live demo section 😨 Whenever you would have this section then you can add this link again in the Table of Contents 👍 to make your readme descriptive and professional 💯